src/app/collective-rewards/metrics/hooks/useGetCycleNext.ts

```diff
@@ -1,28 +0,0 @@
-import { CycleTimeKeeperAbi } from '@/lib/abis/v2/CycleTimeKeeperAbi'
-import { BackersManagerAddress } from '@/lib/contracts'
-import { AVERAGE_BLOCKTIME } from '@/lib/constants'
-import { useQuery } from '@tanstack/react-query'
-import { readContract } from 'wagmi/actions'
-import { config } from '@/config'
-
-export const useGetCycleNext = (timestamp: bigint) => {
-  const { data, isLoading, error } = useQuery({
```

Comment:
By removing the useQuery, we changed the behavior and will generate additional re-renders, we need to pass down this behavior to the `CycleContext`

Reply:
should be fixed now